composer.json

```diff
@@ -5,6 +5,7 @@
     "require": {
         "symfony/config": "^4.4|^5.0",
         "symfony/http-kernel": "^4.4|^5.0",
+        "symfony/messenger": "^4.4|^5.0",
```

Comment:
I think we could move this into dev as someone might want to use symfony http and not messenger. Notice we do this for the http client: https://github.com/jcchavezs/zipkin-instrumentation-symfony/pull/69/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R17

Reply:
Ok